src/WebJobs.Extensions.ApiHub/Table/ApiHubTableAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute that specifies an ApiHub table or entity.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ApiHubTableAttribute : Attribute
+    {
+        /// <summary>
+        /// Gets the key of the configuration setting that specifies the connection string.
+        /// </summary>
+        public string Key { get; }
```

Comment:
Please rename this to "Connection". Several of the other attributes are using "ConnectionString" but our plan is to rename those as per https://github.com/Azure/azure-webjobs-sdk-extensions/issues/67